github_stargazers/github.py

```diff
@@ -80,12 +93,26 @@ def __extract_stargazers_from_url(self, url: str) -> typing.List[str]:
         soup: typing.Optional[BeautifulSoup] = self.__get_soup(url)
         h3_components = soup.find_all('h3')
 
+        def _check_hyperlink_component(component: Tag) -> None:
+            hyperlink_component: typing.Optional[Tag] = component.find('a')
+            if not hyperlink_component:
+                raise MissingHyperlinkTagError()
+            if not hyperlink_component.get('href'):
+                raise MissingHrefAttributeError()
+
+        def _extract_username_from_h3(component: Tag) -> typing.Optional[str]:
+            if component.get_text() == self.__MARK_END_OF_STARGAZERS:
```

Comment:
mmmmm It's weird. You should be able to query for a param like `hasNextPage` and check what's that value instead of comparing the string response.

Reply:
Addressed the above comment [here](https://github.com/marius92mc/github-stargazers/pull/39#issuecomment-343611424).